deltachat-ffi/deltachat.h

```diff
@@ -3287,19 +3214,6 @@ uint32_t        dc_msg_get_from_id            (const dc_msg_t* msg);
 uint32_t        dc_msg_get_chat_id            (const dc_msg_t* msg);
 
 
-/**
- * Get the ID of chat the message belongs to.
- * To get details about the chat, pass the returned ID to dc_get_chat().
- * In contrast to dc_msg_get_chat_id(), this function returns the chat-id also
- * for messages in the deaddrop.
- *
- * @memberof dc_msg_t
- * @param msg The message object.
- * @return The ID of the chat the message belongs to, 0 on errors.
- */
-uint32_t        dc_msg_get_real_chat_id       (const dc_msg_t* msg);
```

Comment:
to make upgrade easier, i would add a `#define dc_msg_get_real_chat_id(a) dc_msg_get_chat_id((a))` , cmp comment to removed DC_CHAT_ID_DEADDROP

Reply:
Same as above, keeping these functions makes it easy to overlook places where changes are needed, as they will compile without warnings and then work incorrectly then.